apps/mull-ui/src/app/pages/create-event/create-event.scss

```diff
@@ -92,13 +93,29 @@
     padding: 1em 2.5em 1em 1em;
     margin-bottom: 0.275rem;
     width: 100%;
+    margin-top: 0.5rem;
     box-sizing: border-box;
     &.error {
       border-color: $error-color;
     }
   }
 }
 
+.pill-options-container {
+  width: 100%;
+  margin-top: 1rem;
+  .pill-options {
+    width: 100%;
+    margin-top: 0.5rem;
+    button:nth-of-type(2) { // remove border around second pill
+      border-right: 0;
+      border-left: 0;
+    }
+    .pill-option {
+      width: 100%;
+    }
+  }
```

Comment:
💡 This styling should be integrated into `pill-options.scss`, since it's changing the way the insides of the component are styled.

In general, if you're changing the insides of the styling of a component, the component itself should be responsible, not its parent.

Reply:
oh you're right, I will move it now